cg/cli/analysis.py

```diff
@@ -12,14 +12,16 @@
 LOG = logging.getLogger(__name__)
 PRIORITY_OPTION = click.option('-p', '--priority', type=click.Choice(['low', 'normal', 'high']))
 EMAIL_OPTION = click.option('-e', '--email', help='email to send errors to')
+START_WITH_PROGRAM = click.option('-sw', '--startwith', help='start mip-pipeline with this program and run downstream processes')
```

Comment:
I would prefer to name this `--start-with`. All words separated by a `-`. This will translate to a `start_with` variable name in python.